packages/x64dbg.dbgchild.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,61 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} 'x64dbg\release' -Resolve
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        unzipLocation = $toolSrcDir
+        url           = 'https://github.com/therealdreg/DbgChild/releases/download/beta10/DbgChild.Beta.10.zip'
+        checksum      = 'f17f588795d8f5f94d71335a8acfa58946bb03a94a5637be7f3e804c652ea2b4'
+        checksumType  = 'sha256'
+    }
+
+    VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+    Install-ChocolateyZipPackage @packageArgs
+    $toolSrcDir = Join-Path $toolSrcDir 'DbgChild Beta 10'
+    VM-Assert-Path $toolSrcDir
+
+    $archs = @("x32", "x64")
+    foreach ($arch in $archs) {
+        $archDstDir = Join-Path $toolDir "${arch}" -Resolve
+        $pluginDstDir = Join-Path $archDstDir 'plugins'
+        if (-Not (Test-Path $pluginDstDir -PathType Container)) {
+            New-Item -ItemType directory $pluginDstDir -Force -ea 0 | Out-Null
+        }
+        VM-Assert-Path $pluginDstDir
```

Comment:
[nitpick] I think this is not needed as you are using `-Force` before

Reply:
I may be wrong, as I am not a PowerShell expert. But I think there might still be scenarios that cause the directory to not be created? My reading of `-Force` when creating directories is that if the directory exists this will return the existing directory instead of erroring out, not that it guarantees the directory will then exist. It seems to help in cases like this: 

```
PS C:\Users\flare > New-Item -ItemType directory C:\Users\flare\
New-Item : An item with the specified name C:\Users\flare\ already exists.
At line:1 char:1
+ New-Item -ItemType directory C:\Users\flare\
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceExists: (C:\Users\flare\:String) [New-Item], IOException
    + FullyQualifiedErrorId : DirectoryExist,Microsoft.PowerShell.Commands.NewItemCommand

FLARE-VM 03/15/2024 20:33:15
PS C:\Users\flare > New-Item -ItemType directory C:\Users\flare\ -Force


    Directory: C:\Users


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
d-----          3/7/2024   1:01 PM                flare
```

But it can't help if you give some totally bad path for whatever reason (here D: does not exist):

```
PS C:\Users\flare > New-Item -ItemType directory D:\bad -Force
New-Item : The path is not of a legal form.
At line:1 char:1
+ New-Item -ItemType directory D:\bad -Force
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (D:\:String) [New-Item], ArgumentException
    + FullyQualifiedErrorId : CreateDirectoryArgumentError,Microsoft.PowerShell.Commands.NewItemCommand
```